src/components/LoanCards/ListLoanCard.vue

```diff
@@ -315,8 +324,12 @@ export default {
 				}
 			}
 
+			.list-loan-card-matching-text {
+				margin: 0.4rem 0 0;
+			}
+
 			.list-loan-card-body-info {
-				margin-top: 0;
+				margin-top: 0.5rem;
```

Comment:
ensures some spacing on the horizontal layout when the matching text isn't present.